### PR TITLE
[MIG] portal_partner_select_all

### PR DESCRIPTION
**Pull request by mclaeysb**
Reverse migrated from v12.

### CODE REVIEW

**Comment by mclaeysb** [comment]
See also https://github.com/OCA/partner-contact/pull/744

**Comment by mclaeysb** [comment]
@chienandalu I have difficulties understanding the codecov checks. Could you help me?

**Review by chienandalu — APPROVED** [review]
Hi @mclaeysb Thanks for your PR :) I tested it and it's ok

codecov/patch stands for code inside this PR not covered by tests. You can enter into the details and click on the diff tab to see what that code is.
codecov/project stands for the increment (or decrement) of uncovered code added to the repository.

It's highly recommended to add tests to modules to avoid regressions.

File: portal_partner_select_all/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+When a user wants invite a lot of custumers to the portal it can be a pain to
```
Comment (skeller1) [inline portal_partner_select_all/readme/DESCRIPTION.rst]: typo: `custumers` => `customers`

**Review by skeller1 — CHANGES_REQUESTED** [review]
missing tests

code at all: :+1:  LGTM

Tested it on runbot (after assigning the missing user rights the action was available in UI) :+1:

File: portal_partner_select_all/readme/DESCRIPTION.rst (hunk 1 above)
Comment (mclaeysb) [reply portal_partner_select_all/readme/DESCRIPTION.rst]: Typo fixed. The original code had no tests. Will this be merged without?

Comment (skeller1) [reply portal_partner_select_all/readme/DESCRIPTION.rst]: 12.0 has no tests, so maybe it should be merged too. Thank you for contributing.

**Review by skeller1 — APPROVED** [review]

**Review by skeller1 — CHANGES_REQUESTED** [review]
You have to regenerate the docs (to get new README, new static/description index.html) with changed typo

**Comment by mclaeysb** [comment]
Here we go! Hope this can indeed be merged :)

**Review by skeller1 — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]

**Pull request merged**